src/lib.rs

```diff
@@ -75,6 +77,89 @@ pub enum SymbolKind {
     Tls,
 }
 
+/// An iterator of the sections of an File
+pub struct SectionIterator<'a, 'b> {
+    inner: SectionIteratorInternal<'a, 'b>
+}
+
+impl<'a, 'b> fmt::Debug for SectionIterator<'a, 'b> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        // It's painful to do much better than this
```

Comment:
Hmm yeah, the `Debug` requirement isn't giving much benefit here. I also see that the `Debug` implementation we added for `File` isn't useful.